File: Code/Disney/disney.xBandController/src/linux/xibackfill/src/com/disney/GuestEligibility.java
package com.disney;

import org.codehaus.jackson.annotate.JsonProperty;

public class GuestEligibility {
	@JsonProperty("magicPlusParticipantStatusEffectiveDate") private String magicPlusParticipantStatusEffectiveDate;
	@JsonProperty("links") private Links links;
	@JsonProperty("magicPlusParticipantStatus") private String magicPlusParticipantStatus;
}

File: Code/Disney/disney.xBandController/src/windows/WDW.NGE.Support.Dto/OneView/GuestEligibility.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace WDW.NGE.Support.Dto.OneView
{
    [DataContract]
    public class GuestEligibility
    {
        [DataMember(Name = "magicPlusParticipantStatus", Order = 1)]
        public String MagicPlusParticipantStatus { get; set; }
        
        [DataMember(Name = "magicPlusParticipantStatusEffectiveDate", Order = 2)]
        public String MagicPlusParticipantStatusEffectiveDate { get; set; }

        [DataMember(Name = "links", Order = 3)]
        public GuestEligibilityLinks Links { get; set; }

    }
}
